### PR TITLE
Add a retry for USPS address. Try up to 3 times

### PR DESCRIPTION
**Pull request by bytheway875**

### CODE REVIEW

File: app/models/efile_submission.rb [hunk 1]
```diff
@@ -119,10 +119,15 @@ def previously_transmitted_submission
     previous_submission if previous_submission.last_transition_to("transmitted").present?
   end
 
-  def generate_verified_address
+  def generate_verified_address(i = 0)
     return OpenStruct.new(valid?: true) if verified_address.present?
 
-    address_service = StandardizeAddressService.new(intake)
+    address_service = StandardizeAddressService.new(intake, read_timeout: 1500)
```
Comment (bytheway875) [inline app/models/efile_submission.rb:125]: a timeout of 1.5 seconds is totally arbitrary. We set a one second timeout in the application call for this and it seemed ok, so decided to go for a bit longer here.

File: app/services/standardize_address_service.rb [hunk 2]
```diff
@@ -4,6 +4,9 @@
 # PLEASE do NOT use this service to standardize addresses in bulk (do NOT use in backfills)
 # as that violates the terms of agreement with USPS and will result in access being revoked.
 class StandardizeAddressService
+  attr_accessor :timeout
+  alias_method :timeout?, :timeout
```
Comment (jenny-heath) [inline app/services/standardize_address_service.rb:8]: ohh never seen this: is this so that you can call `timeout` and `timeout?`?

**Review by jenny-heath — APPROVED** [review]
this looks good! seems to cover the case where the API times out (not sure whether we want the story to cover other types of failures? if so that could go in a separate PR though)

File: app/services/standardize_address_service.rb (hunk 2 above)
Comment (bytheway875) [reply app/services/standardize_address_service.rb:8]: yep!

**Pull request merged**